reference/docs-conceptual/samples/Repeating-a-Task-for-Multiple-Objects--ForEach-Object-.md

```diff
@@ -1,17 +1,23 @@
 ---
-ms.date:  06/05/2017
+ms.date:  12/23/2019
 keywords:  powershell,cmdlet
 title:  Repeating a Task for Multiple Objects ForEach Object
 ---
 # Repeating a Task for Multiple Objects (ForEach-Object)
 
-The **ForEach-Object** cmdlet uses script blocks and the `$_` descriptor for the current pipeline object to let you run a command on each object in the pipeline. This can be used to perform some complicated tasks.
+The `ForEach-Object` cmdlet uses script blocks and the `$_` descriptor for the current pipeline
+object to let you run a command on each object in the pipeline. This can be used to perform some
+complicated tasks.
 
-One situation where this can be useful is manipulating data to make it more useful. For example, the Win32_LogicalDisk class from WMI can be used to return free space information for each local disk. The data is returned in terms of bytes, however, which makes it difficult to read:
+One situation where this can be useful is manipulating data to make it more useful. For example, the
+**Win32_LogicalDisk** class from WMI can be used to return free space information for each local
+disk. The data is returned in terms of bytes, however, which makes it difficult to read:
 
+```powershell
+Get-CimInstance -Class Win32_LogicalDisk
 ```
-PS> Get-WmiObject -Class Win32_LogicalDisk
 
+```Output
```

Comment:
This output doesn't match mine from 6.2.3. This returns a table instead of a list by default now.  Either the code or the output should be updated to align.

Reply:
Fixed